workflows/post_process_run/scream_post/format_output.py

```diff
@@ -0,0 +1,186 @@
+from typing import Hashable, List, Mapping
```

Comment:
How would this script get called? Are you imagining some automated post-processing, or more of an ad-hoc approach? Will the script require a lot of memory to run for our anticipated datasets?

For now, this script may be better suited for the `fv3net/projects` location, which is a good place for untested code. Everything else in `post_process_run` is quite well unit tested and more modular so I'm not sure this script is currently a good fit.

Reply:
Moved to `fv3net/projects/scream`. This script is called once offline to transform scream native output to zarr, it also includes some hacks to work with the current set of scream output. We expect to remove these hacks (e.g., surface pressure is 2x, horizontal winds are written in the same variable but will be split into 2) once we have the new scream output. Wanted to still commit this so that the dataset is reproducible